interface/super/manage_site_files.php

```diff
@@ -122,6 +122,56 @@
 }
 
 
+/**
+ * White list files.
+ * Security feature that enable to upload only file with mime-type from white list.
+ * Important to prevention upload of virus script.
+ * Dependence - turn on global setting 'secure_upload'
+ */
+
+if($GLOBALS['secure_upload']){
+
+    $mime_types  = array('image/*', 'text/*', 'audio/*', 'video/*');
+
+    // Get cURL resource
+    $curl = curl_init();
+
+    curl_setopt_array($curl, array(
+        CURLOPT_RETURNTRANSFER => 1,
+        CURLOPT_URL => 'https://cdn.rawgit.com/jshttp/mime-db/master/db.json'
```

Comment:
I can put hard-coded list (partial..) for such cases.. anyway there is option to add mimetype to the list manually.